charts/ocis/templates/_common/images.tpl

```diff
@@ -0,0 +1,66 @@
+{{/*
+imagePullSecrets logic
+*/}}
+{{- define "ocis.imagePullSecrets" -}}
+  {{- with $.Values.image.pullSecrets }}
+imagePullSecrets:
+  {{- toYaml . | nindent 2 }}
+  {{- end }}
+{{- end -}}
+
+{{/*
+oCIS image logic
+*/}}
+{{- define "ocis.image" -}}
+  {{ $tag := default .Chart.AppVersion .Values.image.tag -}}
+  {{ if $.Values.image.sha -}}
```

Comment:
I would unify the handling for `sha` in all template. So either have a variable and use that (my preference) or without a variable. Currently we have a mixture.

Reply:
Really good solution. Like it.

Reply:
> Really good solution. Like it.

Then I'll merge this.